src/preparation.jl

```diff
@@ -48,27 +51,31 @@ function roll_up_chunks(parsed_blocks, url::AbstractString; separator::String="<
     str = String(take!(io))
     if !isempty(str)
         push!(docs, str)
-        src = url * (isnothing(last_header) ? "" : "::$last_header")
+        src = url * (isnothing(last_header) ? "" : " - $last_header")
         push!(sources, src)
     end
     return docs, sources
 end
 
 
 struct DocParserChunker <: RT.AbstractChunker end
-"""
-    RT.get_chunks(chunker::DocParserChunker,
-    html_files::Vector{<:AbstractString};
-    sources::AbstractVector{<:AbstractString}=html_files,
-    verbose::Bool=true,
-    separators=["\n\n", ". ", "\n", " "], max_length::Int=256)
 
-Extracts chunks from HTML files, by parsing the content in the HTML, rolling up chunks by headers, and splits them by separators to get the desired length.
+"""
+    RT.get_chunks(chunker::DocParserChunker, url::AbstractString;
+        verbose::Bool=true, separators=["\n\n", ". ", "\n", " "], max_chunk_size::Int=MAX_CHUNK_SIZE)
+
+Extract chunks from HTML files, by parsing the content in the HTML, rolling up chunks by headers, 
+and splits them by separators to get the desired length.
+
+# Arguments
+- chunker: DocParserChunker
+- url: URL of the webpage to extract chunks
+- verbose: Bool to print the log
+- separators: Chunk separators
+- max_chunk_size Maximum chunk size
 """
 function RT.get_chunks(chunker::DocParserChunker, url::AbstractString;
-    verbose::Bool=true,
-    separators=["\n\n", ". ", "\n", " "], max_length::Int=256)
-
+    verbose::Bool=true, separators=["\n\n", ". ", "\n", " "], max_chunk_size::Int=MAX_CHUNK_SIZE)
```

Comment:
why do you introduce new kwarg MAX_CHUNK_SIZE? Why not calling the same as all other RAG, or was it confusing? (it's harder for users who know normal airag)

Reply:
It's okay as long as you document it properly

Reply:
if `max_length` is a standardized variable then I'll use the same. I thought it might be confusing